src/applications/personalization/dashboard-2/components/claims-and-appeals/Claim.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import moment from 'moment';
+
+import recordEvent from '~/platform/monitoring/record-event';
+import {
+  getPhaseDescription,
+  isClaimComplete,
+  getClaimType,
+} from '~/applications/claims-status/utils/helpers';
+
+const capitalizeFirstLetter = input => {
+  const capitalizedFirstLetter = input[0].toUpperCase();
+  return `${capitalizedFirstLetter}${input.slice(1)}`;
+};
+
+function listPhase(phase) {
```

Comment:
a lot of this component's code was pulled from https://github.com/department-of-veterans-affairs/vets-website/blob/e9d31626da74cb50e708760661b7935cfa44b7af/src/applications/claims-status/components/appeals-v2/ClaimsListItemV2.jsx